frontend/src/components/Copy.tsx

```diff
@@ -1,4 +1,6 @@
+import { toast } from "react-hot-toast"
 import { ReactComponent as CopyIcon } from "../images/icons/copy.svg"
+import MutinyToaster from "./MutinyToaster"
```

Comment:
this means there will be two mutinytoasters on some pages... but it doesn't seem to cause double toasts so no harm.

we can probably end up with one global mutinytoaster at some point but this is fine for now